queries/ruby.scm

```diff
@@ -13,3 +13,4 @@
 (regex) @regularExpression
 
 (class) @class
+(class name: (_) @className) @domain
```

Comment:
Should this shorthand be `@_.domain` instead to make it clearer that it's a related node rather than a different scope type?